guide/payments/send.md

```diff
@@ -39,169 +103,255 @@ layout = "full-width"
 
-If your application allows setting spending limits, and the current transaction exceeds it, make sure they go through some security check (biometric, enter a PIN, 2FA password, etc). This technique can also be employed if the transaction is attempting to use the max wallet balance.
 </div>
 
-**Do's**
+## Errors
 
-- Allow the sender to review payment details like address, amount, and fees and adjust if necessary before submitting the transaction
-- Show the amount being sent and the fee information in both bitcoin and the senders preferred currency
+Handling problems gracefully is particularly important when it comes to payments, as users may be concerned about having lost funds. Clear communication may include:
+- A brief explanation of what went wrong
+- Status of the funds and the payment as a whole
+- How the problem can be fixed or avoided
+- What to do next
 
-## Transaction processing and confirmation
+<div class="image-slide-gallery">
 
-{% include image.html
-   image = "/assets/images/guide/payments/send/first-confirmation.png"
-   retina = "/assets/images/guide/payments/send/first-confirmation@2x.png"
-   mobile = "/assets/images/guide/payments/send/first-confirmation-mobile.png"
-   mobileRetina = "/assets/images/guide/payments/send/first-confirmation-mobile@2x.png"
-   alt-text = "Notifications after the transaction is broadcast"
-   width = 800
-   height = 270
+{% for item in page.imagesErrors %}
+
+{% capture imageURL %}{{ page.image_base }}{{ item.file }}.png{% endcapture %}
+{% capture imageURLRetina %}{{ page.image_base }}{{ item.file }}@2x.png{% endcapture %}
+
+{% include picture.html
+   image = imageURL
+   retina = imageURLRetina
+   alt-text = item.alt
+   caption = item.caption
+   width = 250
+   height = 541
+   layout = "shadow"
 %}
+{% endfor %}
+
+</div>
+
+It is ideal when the application can automatically identify and fix or avoid the problem. This may not always be possible, or even wanted if the solution incurs extra fees or takes time. For practical purposes, it can be helpful to group problems into categories, such as:
+
+- The problem cannot be identified
+- The problem can be identified and requires manual action by the user
+- The problem can be automatically fixed, but the user needs to choose which solution to use
+- The app can likely automatically fix the problem with negligible impact on the user
+
+Effectively supporting users when problems occur can build trust and confidence, and essential aspect for financial applications.
+
+---
 
-Let us look at how we communicate to the sender about the [processing of a transaction]({{ '/guide/payments/transactions/#transaction-lifecycle' | relative_url }}) after it has been broadcast. There are three main states that you would want to inform or notify the sender of:
+## Advanced options
```

Comment:
Some things you could add here:

- RBF
- Let users choose payment hops
- Enable an AMP payment (this should be by default though, might want to add something about this in this section as it's a major UX improvement).

Reply:
RBF is already mentioned, or do you mean I should provide more detail (like a mock-up)?

By payment hops, is that the discussion around probability of success and fees? Or what am I choosing?

AMP seems like an under-the-hood thing, so I only included it in the format sub-page. Ideally, the wallet can pick the most appropriate technique for making the payment/transaction.